### PR TITLE
C73 design export html

### PR DESCRIPTION
**Pull request by TheOneHoff**
## Change Summary

- Redesigned the export page
- User can now choose from some pre-selected time frames (last week, last day, etc.)

### Change Form
**Fill this up (NA if not available). If a certain criteria is not met, can you please give a reason.**

- [x] The pull request title has an issue number
- [x] The code works by "Smoke testing" (in other words, you have tested it your own and it works)
- [ ] The code has documentation (either comments or in main documentation platform)

## Other Information
**[Is there anything in particular in the review that I should be aware of?]**

### CODE REVIEW

**Review by JordanHartley — APPROVED** [review]
All looks good great job!

File: app/app/static/custom/ourscript.js [hunk 1]
```diff
@@ -352,10 +352,45 @@ window.onload = async (event) => {
...
+			start.setMonth(start.getMonth()-1)
+			startDate = start.getFullYear()+'-'+('0' + (start.getMonth()+1)).slice(-2)+'-'+start.getDate();
+			break;
+		case 'Last Year':
+			start.setFullYear(start.getFullYear()-1)
+			startDate = start.getFullYear()+'-'+('0' + (start.getMonth()+1)).slice(-2)+'-'+start.getDate();
+			break;
+		case 'All Time':
+			startDate = firstDay;
+			break;
```
Comment (frinzekt) [inline app/app/static/custom/ourscript.js:384]: consider using Javascript string template for this

**Review by frinzekt — APPROVED** [review]
looks good! merge whenever you are ready

File: app/app/static/custom/ourscript.js (hunk 1 above)
Comment (frinzekt) [reply app/app/static/custom/ourscript.js:384]: or maybe even using the iso string and splice it? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toISOString

I am sure there must be a standard built-in way to get the datestring

Comment (TheOneHoff) [reply app/app/static/custom/ourscript.js:384]: wasn't aware of the toISOstring method. That makes it much cleaner

**Pull request merged**